modules/web/.eslintrc.yml

```diff
@@ -62,7 +62,7 @@ rules:
   prefer-rest-params: error
   complexity:
     - error
-    - 15
+    - 20
```

Comment:
@Waseem826 It's better to reduce complexity in your function instead of increasing this in eslint config.

Reply:
I'll add ts-ignore to mute this error because its not possible to reduce complexity in my scenario

Reply:
I'll add ts-ignore to mute this error because its not possible to reduce complexity in my scenario

Reply:
Can't you use a map instead of using switches?

Reply:
Yea, ended up using map.